packages/coachmark/test/coach-indicator.test.ts

```diff
@@ -30,14 +30,6 @@ describe('CoachIndicator', () => {
         await elementUpdated(el);
         await expect(el).to.be.accessible();
     });
-    it('loads coach-indicator white static-color when static is set', async () => {
-        const el = await fixture<CoachIndicator>(html`
-            <sp-coach-indicator static="white"></sp-coach-indicator>
-        `);
-        await elementUpdated(el);
-        expect(el.staticColor == 'white').to.be.true;
-        expect(el.getAttribute('static-color')).to.equal('white');
-    });
     it('loads coach-indicator white static-color variant', async () => {
         const el = await fixture<CoachIndicator>(html`
             <sp-coach-indicator variant="white"></sp-coach-indicator>
```

Comment:
no variant now right?

Reply:
This PR was focused on the `static` references only. The other deprecations were handled in the individual PRs.